packages/tools/src/stateManagement/segmentation/SegmentationStateManager.ts

```diff
@@ -752,9 +749,9 @@ export default class SegmentationStateManager {
   }
 }
 
-async function computeVolumeSegmentationFromStack({
+async function internalComputeVolumeLabelmapFromStack({
   imageIds,
-  options,
+  options = {},
```

Comment:
Make this optional rather than defining a default value, as line 766 below allows for optionality.  Also, the volumeId should be || not ??